notesystem/modes/convert_mode.py

```diff
@@ -156,7 +161,8 @@ class Handler(FileSystemEventHandler):
             def on_any_event(self, event: FileSystemEvent):
                 if event.is_directory:
                     return None
-                elif event.event_type == 'created' or event.event_type == 'modified':
+                elif event.event_type == 'created' \
+                        or event.event_type == 'modified':
```

Comment:
Is this the nicest way this can be done?
```suggestion
                elif ( event.event_type == 'created' or
                        event.event_type == 'modified'):
```